interface/super/show_groups_popup.php

```diff
@@ -48,8 +48,11 @@
 <h1><?php xl('Layout groups','e'); ?></h1>
 <ul>
 <?php
-$res = sqlStatement("SELECT distinct(group_name) FROM layout_options WHERE " .
-                    "form_id = '".$_GET['layout_id']."' ORDER BY group_name");
+
+$query .= "SELECT distinct(group_name) FROM layout_options WHERE ";
+$query .= "form_id = ? ORDER BY group_name";
+$res = sqlStatement($query, array($_GET['layout_id']));
+
```

Comment:
LGTM i think this is ok just a ques, y the split of the `$query`. Cant it be on the same line? @prondubuisi

Reply:
We need to think coding style henceforth

Reply:
made the change @muarachmann thanks

Reply:
okay looks cleaner now @prondubuisi thanks